lib/matplotlib/projections/geo.py

```diff
@@ -190,7 +190,7 @@ def set_longitude_grid(self, degrees):
         """
         Set the number of degrees between each longitude grid.
         """
-        number = (360.0 / degrees) + 1
+        number = int(360.0 / degrees) + 1
```

Comment:
Actually, I don't think this line is correct. It should probably be  `number =  int(np.round(360 / degrees))`. Right now, if degrees = 360, then number is equal to 2, which I am pretty sure is wrong.